eth/client.go

```diff
@@ -361,12 +361,8 @@ func (c *client) Account() accounts.Account {
 	return c.accountManager.Account()
 }
 
-func (c *client) Backend() (Backend, error) {
-	if c.backend == nil {
-		return nil, ErrMissingBackend
-	} else {
-		return c.backend, nil
-	}
+func (c *client) Backend() Backend {
+	return c.backend
```

Comment:
Dropped the error in the return type for this method because there should always be a backend configured when a client is instantiated and doing so makes it easier to call backend methods when the caller only has access to the client i.e. we can do `client.Backend().SomeMethod()` without additional error handling.